examples/css/index.html

```diff
@@ -83,6 +83,30 @@ <h3 class="mistica-text-title2">Tag</h3>
                     <div class="mistica-tag-error">Error</div>
                 </div>
 
+                <h3 class="mistica-text-title2">TextFields</h3>
+                <h4 class="mistica-text-title1">TextField</h4>
+                <div class="vertical-group">
+                    <div class="mistica-text-field">
+                        <input id="text-field-1" type="text" placeholder=" " />
```

Comment:
note that `placeholder` is always needed. Without a placeholder we can't implement the shrinking label effect when the input is filled. With placeholder we can use the `:placeholder-visible` pseudoclass to do the trick. When we don't want the input to have a visible placeholder, we can set it to `" "` as I'm doing in this example